parm/use_cases/met_tool_wrapper/PointStat/PointStat.conf

```diff
@@ -103,6 +103,7 @@ POINT_STAT_OUTPUT_FLAG_VL1L2 = STAT
 #POINT_STAT_CLIMO_CDF_BINS = 1
 #POINT_STAT_CLIMO_CDF_CENTER_BINS = False
 #POINT_STAT_CLIMO_CDF_WRITE_BINS = True
+#POINT_STAT_CLIMO_CDF_DIRECT_PROB =
```

Comment:
Why do we have commented-out defaults listed for all the other CLIMO_CDF options, but no the new DIRECT_PROB one?

Reply:
See comment in GridStat.conf